src/main/java/io/kestra/plugin/serdes/xml/XmlReader.java

```diff
@@ -32,6 +33,7 @@
 @Schema(
     title = "Read a XML file and write it to an ion serialized data file."
 )
+@Slf4j
```

Comment:
Can you fix the query property doc (even though out of scope it doesn't take much)